README.md

```diff
@@ -74,19 +74,16 @@ process.mainArgs = process.argv.slice(process._exec ? 1 : 2)
 
 ----
 
-## 💡 `util.parseArgs([argv][, options])` Proposal
+## 💡 `util.parseArgs([config])` Proposal
 
-* `argv` {string[]} (Optional) Array of argument strings; defaults
-  to [`process.mainArgs`](process_argv)
-* `options` {Object} (Optional) The `options` parameter is an
+* `config` {Object} (Optional) The `config` parameter is an
   object supporting the following properties:
-  * `withValue` {string[]} (Optional) An `Array` of argument
-    strings which expect a value to be defined in `argv` (see [Options][]
-    for details)
-  * `multiples` {string[]} (Optional) An `Array` of argument
-    strings which, when appearing multiple times in `argv`, will be concatenated 
-into an `Array`
-  * `short` {Object} (Optional) An `Object` of key, value pairs of strings which map a "short" alias to an argument; When appearing multiples times in `argv`; Respects `withValue` & `multiples`
+  * `argv` {string[]} (Optional) Array of argument strings; defaults
+    to [`process.mainArgs`](process_argv)
+  * `options` {Object} (Optional) An object describing the known options to look for in `argv`; `options` keys are the long names of the known options, and the values are objects with the following properties:
```

Comment:
> `options` keys are the long names of the known options

Does this mean I can not add configuration for an option which only has a short form?

(I do not consider this a blocker, and can be solved. Just making sure people are aware of the new limitation.)

Reply:
> Does this mean I can not add configuration for an option which only has a short form?

No you can. Just pass a single character as the long name: e.g.
```js
// node long-short.js -f bar
const {options} = parseArgs({
  options: {
    f: {
      type: 'string'
    }
  }
})
options // { f: 'bar' }
```

Reply:
I think that will add configured `--f` but not `-f`. In particular, `-f` will not be found by this code:

```
        arg = StringPrototypeCharAt(arg, 1); // short
        for (const [option, optionConfig] of ObjectEntries(options)) {
          if (optionConfig.short === arg) {
            arg = option; // now long!
            break;
          }
```

Reply:
I misunderstood the implementation, I'll add a new comment and hide the above.

Reply:
A quirk of the current implementation is that a one character long option will be recognised as both a long option (e.g. `--f`) and a short option (e.g. `-f`). This is an existing quirk, and not introduced by this PR.

Reply:
I see no issue with this unless we think there are cases where authors do NOT want to allow shorts for a given option. Is this a scenario you've encountered maintaining Commander? Or foresee other problems with this quirk?

Reply:
Commander started out with dual short and long option names (needed to define both). Support for long-only options added in 2011. Support for short-only options added in 2020 (and more for completeness than because of demand).